sirepo/job_driver/__init__.py

```diff
@@ -201,6 +221,7 @@ def pkdebug_str(self):
         Returns:
             bool: True if the op was actually sent
         """
+        self.ops[op.opId] = op
```

Comment:
I don't understand why this was added here.

Reply:
It was moved from `op_ready` to here. I did this because the call to `await self._agent_start(op)` ([line 223](https://github.com/radiasoft/sirepo/blob/issue/2245-agent-start-fail-noisily/sirepo/job_driver/__init__.py#L223)) is where we will block when the agent fails to start. In the code to handle the failed start we destroy all ops in self.ops and in the old code self.ops wouldn't contain op.